modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,46 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-interacting-with-pipelines-using-the-developer-perspective_{context}"]
+= Interacting with Pipelines using the Developer perspective
+
+The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a project along with details, such as the namespace in which the Pipeline was created, the last PipelineRun, the status of the Tasks in the PipelineRun, the status of the PipelineRun, and the time taken for the run.
+
+[Discrete]
+.Procedure
+. In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down to see the Pipelines in that project.
++
+//image::op-pipeline-list.png[Pipelines in the Developer perspective]
++
+. Click on the required Pipeline to see the *Pipeline Details* page. It provides a visual representation of all the serial and parallel Tasks in the Pipeline. The Tasks are also listed at the bottom-right of the page. You can click the listed *Tasks* to view the details of the Task.
```

Comment:
s/to view the details of the Task/to view task details/

Reply:
done